apps/web/composables/usePaymentMethods/usePaymentMethods.ts

```diff
@@ -26,8 +26,15 @@ export const usePaymentMethods: UsePaymentMethodsReturn = () => {
   const fetchPaymentMethods: FetchPaymentMethods = async () => {
     state.value.loading = true;
     const { data, error } = await useAsyncData(() => useSdk().plentysystems.getPaymentProviders());
+
+    const klarnaPaymentKeys = ['klarna', 'klarnasliceit', 'klarnapaylater', 'klarnapaynow'];
```

Comment:
wouldn't it make more sense to do this after the error handling? where state.value.data is already set?